tinyagentos/routes/shortcut_proxy.py

```
@@ -0,0 +1,488 @@
+    await websocket.accept()
+
+    try:
+        import websockets as _ws_lib
+        async with _ws_lib.connect(upstream_ws_url) as upstream:
+            async def _client_to_upstream():
+                try:
+                    while True:
+                        data = await websocket.receive_bytes()
+                        await upstream.send(data)
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)